core/controllers/acl_decorators.py

```diff
@@ -1754,12 +1756,13 @@ def can_edit_story(handler):
             a user has permission to edit a story for a given topic.
     """
 
-    def test_can_edit_story(self, topic_id, **kwargs):
+    def test_can_edit_story(self, story_id, *args, **kwargs):
```

Comment:
Do you still need to add *args, here and elsewhere? If not, please remove *args from decorators that do not need them.

Reply:
Done, removed.